webapp-django/crashstats/signature/views.py

```diff
@@ -127,16 +135,27 @@ def signature_reports(request, params):
 
     context['params'] = current_query.copy()
 
+    if '_sort' in context['params']:
+        del context['params']['_sort']
+
     if '_columns' in context['params']:
         del context['params']['_columns']
 
+    context['sort'] = request.GET.getlist('_sort') or DEFAULT_SORT
```

Comment:
In `pass_validated_params` you validate this list. (by making sure it's in `fields`). Here you don't. I don't think that's an inroad to XSS because the `context['sort']` in this case is never printed in HTML. 
But I don't know what craziness a hacker could trigger by sorting on something we don't first check to be a valid field. [For example](http://socorro.dev/search/results/?product=Firefox&version=50.0a1&_sort=CRAZYCRAZYCRAZY&_sort=date&_sort=product&_facets=signature&_columns=date&_columns=signature&_columns=product&_columns=version&_columns=build_id&_columns=platform).